### PR TITLE
Fix issue with serving assets from memory

### LINKED ISSUE #123: Fix issue with serving assets from memory

### PR DESCRIPTION
**Pull request by ganemone**
Fixes #123

### CODE REVIEW

File: test/cli/dev.js [hunk 1]
```diff
@@ -21,23 +22,37 @@ test('`fusion dev` works', async t => {
   t.end();
 });
 
-test('`fusion dev` works with assets', async t => {
+test.only('`fusion dev` works with assets', async t => {
```
Comment (mlmorg) [inline test/cli/dev.js]: Remove .only

Comment (ganemone) [reply test/cli/dev.js]: good catch!

Comment (KevinGrandon) [reply test/cli/dev.js]: (We should implement some sort of linter against this probably)

**Comment by ganemone** [comment]
!merge

**Pull request merged**